bin/check-openpmix.py

```diff
@@ -114,7 +116,7 @@ def check_missing_openpmix(std_all_refs, openpmix_all_refs,
                         possible_error = True
 
                 if possible_error is True:
-                    sys.exit(1)
+                    continue
```

Comment:
Don't we still want this to error out? This indicates that something is wrong with the script so the results are suspect.

Reply:
Ditto here - just wasn't clear to me that this was an error in the script.